RequestReview/action.yml

```diff
@@ -10,6 +10,10 @@ inputs:
   jira-token:
     description: 'Jira auth Token'
     required: true
+  is-infra:
```

Comment:
The name might change. I'm checking with Julien Carsique what would be a suitable time-resistant name